packages-exp/auth-types-exp/index.d.ts

```diff
@@ -327,152 +674,447 @@ export abstract class EmailAuthProvider implements AuthProvider {
 }
 
 /**
- * https://firebase.google.com/docs/reference/js/firebase.auth.multifactorsession
+ * The multi-factor session object used for enrolling a second factor on a user or helping sign in 
+ * an enrolled user with a second factor.
+ * 
+ * @public
  */
 export interface MultiFactorSession {}
 
 /**
- * https://firebase.google.com/docs/reference/js/firebase.user.multifactoruser
+ * This is the interface that defines the multi-factor related properties and operations pertaining 
+ * to a {@link User}.
+ * 
+ * @public
  */
 export interface MultiFactorUser {
+  /** Returns a list of the user's enrolled second factors. */
   readonly enrolledFactors: MultiFactorInfo[];
+  /**
+   * Returns the session identifier for a second factor enrollment operation. This is used to 
+   * identify the current user trying to enroll a second factor.
+   * 
+   * @returns The promise that resolves with the {@link MultiFactorSession}.
+   */
   getSession(): Promise<MultiFactorSession>;
+  /**
+   * 
+   * Enrolls a second factor as identified by the {@link MultiFactorAssertion} for the current 
+   * user. On resolution, the user tokens are updated to reflect the change in the JWT payload. 
+   * Accepts an additional display name parameter used to identify the second factor to the end 
+   * user. Recent re-authentication is required for this operation to succeed. On successful 
+   * enrollment, existing Firebase sessions (refresh tokens) are revoked. When a new factor is 
+   * enrolled, an email notification is sent to the user’s email.
+   * 
+   * @example
+   * ```
+   * const multiFactorUser = multiFactor(firebase.auth().currentUser);
+   * const multiFactorSession = await multiFactorUser.getSession();
+   * // Send verification code
+   * const phoneAuthProvider = new PhoneAuthProvider();
+   * const phoneInfoOptions = {
+   *   phoneNumber: phoneNumber,
+   *   session: multiFactorSession
+   * };
+   * const verificationId = await phoneAuthProvider.verifyPhoneNumber(phoneInfoOptions, appVerifier);
+   * // Store verificationID and show UI to let user enter verification code.
+   * 
+   * const phoneAuthCredential = PhoneAuthProvider.credential(verificationId, verificationCode);
+   * const multiFactorAssertion = PhoneMultiFactorGenerator.assertion(phoneAuthCredential);
+   * await multiFactorUser.enroll(multiFactorAssertion);
+   * // Second factor enrolled.
+   * ```
+   * 
+   * @param assertion - The multi-factor assertion to enroll with.
+   * @param displayName - The display name of the second factor.
+   */
   enroll(
     assertion: MultiFactorAssertion,
     displayName?: string | null
   ): Promise<void>;
+  /**
+   * Unenrolls the specified second factor. To specify the factor to remove, pass a 
+   * {@link MultiFactorInfo} object (retrieved from `enrolledFactors()`) or the factor's UID 
+   * string. Sessions are not revoked when the account is downgraded. An email notification is 
```

Comment:
When what is downgraded? This sounds like the user's account (whatever that means) rather than the developer's project

Reply:
fixed to say unenrolled